Example/ScanDemo 2/Pods/CML_SCAN/License.md

```diff
@@ -0,0 +1 @@
+//Put the license details here.
```

Comment:
Can you actually put the license here? If you leave this file out entirely, what happens?